lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -540,7 +540,10 @@ void HdVP2Mesh::Sync(
     }
 
     if (HdChangeTracker::IsExtentDirty(*dirtyBits, id)) {
-        _sharedData.bounds.SetRange(delegate->GetExtent(id));
+        // Do not trust GPrim extents until we add code to restore the integrity of the data model
```

Comment:
Looks good. Please also test the bounding box display mode in the 3d view.

Reply:
Maybe better to add a doc here saying VP2 will compute the bounds when none is provided.

Reply:
Second thought: bounding box display mode might be broken by this change.

Reply:
If it is broken, we can keep storing the extent in the shared data, but each draw item can opt in/out on using it.

Reply:
Bounding box was indeed broken (as well as selection when switching to wireframe using wrong BB). This should fix it.